csvcubed/csvcubed/cli/inspect/metadataprinter.py

```diff
@@ -114,22 +139,89 @@ def gen_codelist_info_printable(self) -> str:
 
         return f"- The {self._get_type_str()} has the following code list information:{result.output_str}"
 
-    def gen_head_tail_printable(self) -> str:
+    def gen_dataset_observations_info_printable(self) -> str:
         """
         Generates a printable of top 10 and last 10 records.
 
         Member of :class:`./MetadataPrinter`.
 
         :return: `str` - user-friendly string which will be output to CLI.
         """
-        return ""
 
-    def gen_val_count_printable(self) -> str:
+        result_dataset_url: DatasetURLResult
+        if self.csvw_type == CSVWType.QbDataSet:
+            result_dataset_url = select_qb_dataset_url(
+                self.csvw_metadata_rdf_graph, self.dataset_uri
+            )
+        elif self.csvw_type == CSVWType.CodeList:
+            result_dataset_url = select_codelist_dataset_url(
+                self.csvw_metadata_rdf_graph
+            )
+        else:
+            raise Exception("The input csvw json-ld is not supported.")
```

Comment:
Could you define an new exception class which extends from `Exception` and holds a URL (e.g. `http://purl.org/csv-cubed/err/dupe-col`) which is printed to the user when the exception is caught at the top level of the CLI? 

If each type of exception has its own URL then we can easily generate shortish URLs which a user can follow from their terminal to help them understand the error better when it happens.